SpatialGDK/Documentation/content/generating-a-snapshot.md

```diff
@@ -27,4 +31,4 @@ To generate a snapshot, use the **Snapshot** button on the SpatialOS GDK for Unr
 
  This creates a snapshot called `default.snapshot` in `spatial\snapshots`.
 
-You only need to regenerate snapshots when changing the `SnapshotGenerator.SingletonActorClasses` section in `DefaultEditorSpatialGDK.ini`. For more information, see [Singleton Actors]({{urlRoot}}/content/singleton-actors.md).
+You need to regenerate snapshots when adding/removing `SpatialType` classes or when adding/removing stably named replicated actors in your level.
```

Comment:
You need to regenerate snapshots when:
1. Modifying replicated properties or RPCs for any class which schema is generated for
2. Generating schema for a new class
3. Placing new replicated actors into the level
4. Modifying values on replicated actors which are replicated